GherkinSyncTool/DI/GherkinSyncToolModule.cs

```diff
@@ -15,6 +16,7 @@ protected override void Load(ContainerBuilder builder)
             builder.RegisterType<Context>().SingleInstance();
             builder.RegisterModule<TestRailSynchronizerModule>();
             builder.RegisterModule<AzureDevopsSynchronizerModule>();
+            builder.RegisterType<CustomFieldsChecker>().SingleInstance();
```

Comment:
This is TestRail related type. Let's move to TestRailSynchronizerModule?

Reply:
Moved